doc/api/http.md

```diff
@@ -313,8 +313,10 @@ the data is read it will consume memory that can eventually lead to a
 Node.js does not check whether Content-Length and the length of the
 body which has been transmitted are equal or not.
 
-The request implements the [Writable Stream][] interface. This is an
-[`EventEmitter`][] with the following events:
+The request inherits from [Stream][], with two methods like what we have in
+[Writable Stream][]: [`request.write()`][] and [`request.end()`][].
+
+And it will emit the following events:
```

Comment:
This line is unnecessary and can be removed.

Reply:
Are you sure that the whole line should be deleted? I wonder if we don't have the transitional summary to begin with `request`, but just with a sudden of `request's events`?